src/components/KYCWall/BaseKYCWall.tsx

```diff
@@ -23,32 +23,35 @@ import type {BankAccountList, Policy} from '@src/types/onyx';
 import type {PaymentMethodType} from '@src/types/onyx/OriginalMessage';
 import {getEmptyObject} from '@src/types/utils/EmptyObject';
 import viewRef from '@src/types/utils/viewRef';
-import type {AnchorPosition, DomRect, KYCWallProps, PaymentMethod} from './types';
+import type {AnchorPosition, DomRect, KYCWallProps, KYCWallRef, PaymentMethod} from './types';
 
 // This sets the Horizontal anchor position offset for POPOVER MENU.
 const POPOVER_MENU_ANCHOR_POSITION_HORIZONTAL_OFFSET = 20;
 
 // This component allows us to block various actions by forcing the user to first add a default payment method and successfully make it through our Know Your Customer flow
 // before continuing to take whatever action they originally intended to take. It requires a button as a child and a native event so we can get the coordinates and use it
 // to render the AddPaymentMethodMenu in the correct location.
-function KYCWall({
-    addBankAccountRoute,
-    addDebitCardRoute,
-    anchorAlignment = {
-        horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
-        vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
-    },
-    chatReportID = '',
-    children,
-    enablePaymentsRoute,
-    iouReport,
-    onSelectPaymentMethod = () => {},
-    onSuccessfulKYC,
-    shouldIncludeDebitCard = true,
-    shouldListenForResize = false,
-    source,
-    shouldShowPersonalBankAccountOption = false,
-}: KYCWallProps) {
+function KYCWall(
+    {
+        addBankAccountRoute,
+        addDebitCardRoute,
+        anchorAlignment = {
+            horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
+            vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+        },
+        chatReportID = '',
+        children,
+        enablePaymentsRoute,
+        iouReport,
+        onSelectPaymentMethod = () => {},
+        onSuccessfulKYC,
+        shouldIncludeDebitCard = true,
+        shouldListenForResize = false,
+        source,
+        shouldShowPersonalBankAccountOption = false,
+    }: KYCWallProps,
+    ref: ForwardedRef<KYCWallRef>,
```

Comment:
I think we are getting rid of forward refs. Can you add this to the prop itself?